doc/articles/toc.yml

```diff
@@ -256,6 +258,8 @@
               href: features\ElevatedView.md
             - name: Other features
               href: Uno.UI.Toolkit.md
+        - name: Native Frame Navigation
```

Comment:
put this under 'WinUI features', since Frame navigation is a WinUI feature that has some Uno-specific functionality.